frontend/src/app/accountCreation/AccountCreationApp.tsx

```diff
@@ -22,67 +26,77 @@ import { MfaPhoneVerify } from "./MfaPhoneVerify/MfaPhoneVerify";
 import { MfaOktaVerify } from "./MfaOktaVerify/MfaOktaVerify";
 import { MfaGoogleAuthVerify } from "./MfaGoogleAuthVerify/MfaGoogleAuthVerify";
 import { PasswordForm } from "./PasswordForm/PasswordForm";
-
-interface WrapperProps {
-  activationToken: string;
-}
-const AccountCreation404Wrapper: FunctionComponent<WrapperProps> = ({
-  activationToken,
-  children,
-}) => {
-  if (activationToken === undefined) {
-    return <>Loading...</>;
-  }
-  if (activationToken === null) {
```

Comment:
**question:** It looks like the replacement code doesn't cover this 404 case. Did we decide that it wasn't needed?

Reply:
Yep, exactly. Before we'd throw a 404 if you were ever in an unexpected state (which could be as simple as hitting the back button) but with the new status endpoint we know which page to redirect users to. The status endpoint will also hit the `/not-found` page if the user is in an unknown state.